pkg/nss

```diff
@@ -0,0 +1,42 @@
+[mirrors]
+https://ftp.mozilla.org/pub/mozilla.org/security/nss/releases/NSS_3_29_RTM/src/nss-3.29.tar.gz
+
+[vars]
+filesize=7477439
+sha512=0f4dd026b6b32122d8cafa92fa37199b0678f8fef75e375446eddd0cc6ddda1a796e3222caa8bb01b3633911899394d0cb1e4d392880438f68c8ef7290dcb4fa
+pkgver=1
+
+[deps]
+nspr
+zlib
+
+[build]
+patch -p1 < "$K"/nss-3.29-standalone-1.patch
+
+cd nss
+
+make -j1 BUILD_OPT=1 \
+  NSPR_INCLUDE_DIR=/include/nspr \
+  USE_SYSTEM_ZLIB=1 \
+  ZLIB_LIBS=-lz \
+  $([ $(uname -m) = x86_64 ] && echo USE_64=1) \
+  $([ -f /include/sqlite3.h ] && echo NSS_USE_SYSTEM_SQLITE=1)
```

Comment:
i suppose it would be better to just add sqlite to deps, since if we don't NSS just builds its own copy, so later when you need something else that needs sqlite, you have 2 versions lying around.